types/urlencode/index.d.ts

```diff
@@ -0,0 +1,46 @@
+// Type definitions for urlencode 1.1
+// Project: https://github.com/node-modules/urlencode
+// Definitions by: My Self <https://github.com/kimcoder>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface charsetParam {
+	charset: string;
+}
+
+/**
+ * Encode string
+ * @param str The string for encoding.
+ */
+declare function urlencode(str: string, charset?: string): string;
+
+declare namespace urlencode {
+	/**
+	 * Encode string
+	 * @param str The string for encoding.
+	 */
+	function encode(str: string, charset?: string): string;
+
+	/**
+	 * Decode string
+	 * @param encodedString The encoded string.
+	 */
+	function decode(encodedString: string, charset?: string): string;
+
+	/**
+	 * Parse querystring
+	 * @param queryString Querystring
+	 * @param charsetParam The charset for parsing
+	 */
+	function parse(queryString: string, charsetParam: charsetParam): any;
+	interface charsetParam {
+		charset: string;
+	}
+	/**
+	 * Stringify object
+	 * @param obj Query Object
+	 * @param charsetParam The charset for parsing
+	 */
+	function stringify(obj: any, prefix?: charsetParam, charsetParam?: charsetParam): string;
+}
+
+export default urlencode;
```

Comment:
This needs `export =` instead of `export default`. Refer to https://github.com/DefinitelyTyped/DefinitelyTyped#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-defaultexport=`